docs/source/frameworks/lightgbm.rst

```diff
@@ -2,7 +2,8 @@
 LightGBM
 ========
```

Comment:
I assume we also have to update the docs for lightgbm.

Reply:
yup, did a quick clean up, will need to update its docs to our standard